bucket/spicetify-cli.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "2.9.2",
+    "description": "Tool for customizing the Spotify client.",
+    "homepage": "https://spicetify.app/",
+    "license": "LGPL-2.1-only",
+    "notes": "Make sure to set 'spotify_path' to the directory of your Spotify installation in the Spicetify config",
+    "suggest": {
+        "Spotify": "spotify"
```

Comment:
```suggestion
        "Spotify": "extras/spotify"
```

Reply:
Added in https://github.com/ScoopInstaller/Main/pull/3365.